components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/browser/BrowserModel.java

```diff
@@ -388,7 +388,9 @@ private void createGridImages()
      */
     private double getBarSizeInPx(double ratio)
     {
-        if(unitBarLength == null)
+        if (unitBarLength == null
+                || Double.isInfinite(getPixelsSizeX().getValue())
```

Comment:
maybe better to use the same strategy than the one used in 
``EditorUtil.transformPixelsData`` i.e. if  ``BigResult`` is returned then 1